modules/cnv-using-hostpath-provisioner.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-configuring-local-storage-for-vms.adoc
+
+[id="cnv-using-hostpath-provisioner_{context}"]
+= Using the hostpath provisioner to enable local storage
+
+To deploy the hostpath provisioner and enable your virtual machines to use local
+storage, first create a `HostPathProvisioner` custom resource.
+
+.Prerequisites
+
+* Create a backing directory on each node for the PersistentVolumes (PV) created
+by the hostpath provisioner.
+
+* Apply the SELinux context `container_file_t` to the PV
+backing directory on each node. For example:
++
+----
+$ sudo chcon -t container_file_t -R </path/to/backing/directory>
+----
+
+.Procedure
+
+. Create the `HostPathProvisioner` custom resource file. For example:
++
+----
+$ touch hostpathprovisioner_cr.yaml
+----
+
+. Edit the file, ensuring that the `spec.pathConfig.path` value is the directory
+where you want the hostpath provisioner to create PVs. For example:
++
+[source,yaml]
+----
+apiVersion: hostpathprovisioner.kubevirt.io/v1alpha1
+kind: HostPathProvisioner
+metadata:
+  name: hostpath-provisioner
+spec:
+  imagePullPolicy: IfNotPresent
+  pathConfig:
+    path: "</path/to/backing/directory>"
+    useNamingPrefix: "false"
+----
++
+[NOTE]
+====
+If you did not create the backing directory, the provisioner will attempt to create
+it for you. This can cause `Permission denied` errors.
```

Comment:
it can cause permission denied errors IF the user forgets to run the SELinux command. There will be some additional text with regards to a RHCOS 8 worker, as those require a MachineConfig to setup the SELinux. The yaml for an RHCOS worker looks like this:
```yaml
apiVersion: machineconfiguration.openshift.io/v1
kind: MachineConfig
metadata:
  name: 50-set-selinux-for-hostpath-provisioner
  labels:
    machineconfiguration.openshift.io/role: worker
spec:
  config:
    ignition:
      version: 2.2.0
    systemd:
      units:
        - contents: |
            [Unit]
            Description=Set SELinux chcon for hostpath provisioner
            Before=kubelet.service

            [Service]
            ExecStart=/usr/bin/chcon -Rt container_file_t /var/hpvolumes

            [Install]
            WantedBy=multi-user.target
          enabled: true
          name: hostpath-provisioner.service
```

Reply:
@awels I did not make changes related to the StorageClass yet because of my outstanding question, but I did make the other changes if you'd like to review them :)